src/lib/expression/expression_functional.hpp

```diff
@@ -121,6 +121,7 @@ inline detail::unary<AggregateFunction::Min, AggregateExpression> min_;
 inline detail::unary<AggregateFunction::Avg, AggregateExpression> avg_;
 inline detail::unary<AggregateFunction::Count, AggregateExpression> count_;
 inline detail::unary<AggregateFunction::CountDistinct, AggregateExpression> count_distinct_;
+inline detail::unary<AggregateFunction::StdDevSamp, AggregateExpression> stddev_samp_;
```

Comment:
here, too. I'd spell this out (yeah, I know "avg_" isn't either...) or at least make it "std_dev_samp_"